static/js/fieldUpdater.js

```diff
@@ -12,8 +12,18 @@ export default function initialise(options) {
     const loaderElement = containerElement.querySelector('.' + options.prefix + '-loader');
     const errorElement = containerElement.querySelector('.' + options.prefix + '-error');
 
+    const { default: StorageHandler } = await import("./ajaxUpdate.js");
 
     let current_value = options.attribute_value;
+    const storage = new StorageHandler({
+        url: options.submit_url,
+        ifUnmodifiedSince: options.if_unmodified_since,
+        ifMatch: options.if_match,
+        bodyEncode: options.bodyEncode,
+        customHeaders: {
```

Comment:
Not sure if `Accept` header should go under that "customHeaders" bit, as we have a default of `'application/json'` set, so it could just stay with the other  header that's always passed (`X-CSRFToken`).  This is a quibble though not a necessary change.